test/internal/measures_utils_unittest.cc

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#include "../../src/internal/measures_utils.h"
+#include "gtest/gtest.h"
+
+namespace google {
+namespace internal {
+
+TEST(SerializeToPrometheusTextTest, Initialization) {
+  EXPECT_EQ("", SerializeMetricsToPrometheusTextFormat());
```

Comment:
Should we add a test for a non-empty case too?

Reply:
haven't written another test on it yet - I will implement a non-empty case to ensure the function works as expected.

Reply:
added tests.